apps/nsqd/main.go

```diff
@@ -27,21 +26,20 @@ type program struct {
 
 func main() {
 	prg := &program{}
-	// SIGTERM handling is in Start()
-	if err := svc.Run(prg, syscall.SIGINT); err != nil {
+	if err := svc.Run(prg, syscall.SIGINT, syscall.SIGTERM); err != nil {
 		logFatal("%s", err)
 	}
 }
 
 func (p *program) Init(env svc.Environment) error {
 	if env.IsWindowsService() {
 		dir := filepath.Dir(os.Args[0])
-		return os.Chdir(dir)
+		err := os.Chdir(dir)
```

Comment:
... it turns out that go-svc (now?) has this exact code:
https://github.com/judwhite/go-svc/blob/main/svc_windows.go#L69-L76

Reply:
Nice, I'll drop this